2023-12-16 10:48:02 INFO Index: 0...
2023-12-16 10:48:02 INFO Running task: 1_0...
2023-12-16 10:48:02 INFO Test Passed!


2023-12-16 10:48:02 INFO Index: 1...
2023-12-16 10:48:02 INFO Running task: 2_0...
2023-12-16 10:48:02 INFO Test Passed!


2023-12-16 10:48:02 INFO Index: 2...
2023-12-16 10:48:02 INFO Running task: 3_0...
2023-12-16 10:48:02 INFO Test Passed!


2023-12-16 10:48:02 INFO Index: 3...
2023-12-16 10:48:02 INFO Running task: 4_0...
2023-12-16 10:48:02 INFO Test Passed!


2023-12-16 10:48:02 INFO Index: 4...
2023-12-16 10:48:02 INFO Running task: 5_0...
2023-12-16 10:48:02 INFO Test Passed!


2023-12-16 10:48:02 INFO Index: 5...
2023-12-16 10:48:02 INFO Running task: 6_0...
2023-12-16 10:48:03 INFO Test Passed!


2023-12-16 10:48:03 INFO Index: 6...
2023-12-16 10:48:03 INFO Running task: 7_0...
2023-12-16 10:48:03 INFO Test Passed!


2023-12-16 10:48:03 INFO Index: 7...
2023-12-16 10:48:03 INFO Running task: 8_0...
2023-12-16 10:48:03 INFO Test Passed!


2023-12-16 10:48:03 INFO Index: 8...
2023-12-16 10:48:03 INFO Running task: 9_0...
2023-12-16 10:48:03 INFO Compilation Failed!
2023-12-16 10:48:03 INFO run_output: b'  File "/home/ubuntu/test_codellm/unit_tests/chatglm3/c++_python/test.py", line 71\n    return \\"\\"\n            ^\nSyntaxError: unexpected character after line continuation character\n'


2023-12-16 10:48:03 INFO Index: 9...
2023-12-16 10:48:03 INFO Running task: 10_0...
2023-12-16 10:48:03 INFO Test Passed!


2023-12-16 10:48:03 INFO Index: 10...
2023-12-16 10:48:03 INFO Running task: 11_0...
2023-12-16 10:48:03 INFO Test Passed!


2023-12-16 10:48:03 INFO Index: 11...
2023-12-16 10:48:03 INFO Running task: 12_0...
2023-12-16 10:48:03 INFO Runtime Failed!
2023-12-16 10:48:03 INFO run_output: b'Exception: list index out of range\nException: list index out of range\nException: list index out of range\nException: list index out of range\n[-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000] [-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000,-1.000000]\nPassed 1/5 testcases!\nTest Failed!\n'


2023-12-16 10:48:03 INFO Index: 12...
2023-12-16 10:48:03 INFO Running task: 13_0...
2023-12-16 10:48:04 INFO Test Passed!


2023-12-16 10:48:04 INFO Index: 13...
2023-12-16 10:48:04 INFO Running task: 14_0...
2023-12-16 10:48:04 INFO Test Passed!


2023-12-16 10:48:04 INFO Index: 14...
2023-12-16 10:48:04 INFO Running task: 15_0...
2023-12-16 10:48:04 INFO Test Passed!


2023-12-16 10:48:04 INFO Index: 15...
2023-12-16 10:48:04 INFO Running task: 16_0...
2023-12-16 10:48:05 INFO Runtime Failed!
2023-12-16 10:48:05 INFO run_output: b"Exception: local variable 'sum' referenced before assignment\nException: local variable 'sum' referenced before assignment\nException: local variable 'sum' referenced before assignment\nException: local variable 'sum' referenced before assignment\nException: local variable 'sum' referenced before assignment\nPassed 0/5 testcases!\nTest Failed!\n"


2023-12-16 10:48:05 INFO Index: 16...
2023-12-16 10:48:05 INFO Running task: 17_0...
2023-12-16 10:48:05 INFO Test Passed!


2023-12-16 10:48:05 INFO Index: 17...
2023-12-16 10:48:05 INFO Running task: 18_0...
2023-12-16 10:48:06 INFO Test Passed!


2023-12-16 10:48:06 INFO Index: 18...
2023-12-16 10:48:06 INFO Running task: 19_0...
2023-12-16 10:48:06 INFO Test Passed!


2023-12-16 10:48:06 INFO Index: 19...
2023-12-16 10:48:06 INFO Running task: 20_0...
2023-12-16 10:48:06 INFO Test Passed!


2023-12-16 10:48:06 INFO Index: 20...
2023-12-16 10:48:06 INFO Running task: 21_0...
2023-12-16 10:48:06 INFO Test Passed!


2023-12-16 10:48:06 INFO Index: 21...
2023-12-16 10:48:06 INFO Running task: 22_0...
2023-12-16 10:48:06 INFO Test Failed!
2023-12-16 10:48:06 INFO run_output: b'15.000000 2.000000\nTestcase 1 Failed!\n15.000000 5.000000\nTestcase 2 Failed!\n11.000000 2.000000\nTestcase 3 Failed!\n101728172.000000 78054.000000\nTestcase 4 Failed!\n101100183.000000 145109.000000\nTestcase 5 Failed!\nPassed 0/5 testcases!\nTest Failed!\n'


2023-12-16 10:48:06 INFO Index: 22...
2023-12-16 10:48:06 INFO Running task: 23_0...
2023-12-16 10:48:07 INFO Test Failed!
2023-12-16 10:48:07 INFO run_output: b'1.000000 2.000000\nTestcase 1 Failed!\n2.000000 3.000000\nTestcase 2 Failed!\n3.000000 4.000000\nTestcase 3 Failed!\n0.000000 1.000000\nTestcase 4 Failed!\n91.000000 92.000000\nTestcase 5 Failed!\nPassed 0/5 testcases!\nTest Failed!\n'


2023-12-16 10:48:07 INFO Index: 23...
2023-12-16 10:48:07 INFO Running task: 24_0...
2023-12-16 10:48:07 INFO Test Passed!


2023-12-16 10:48:07 INFO Index: 24...
2023-12-16 10:48:07 INFO Running task: 25_0...
2023-12-16 10:48:07 INFO Test Failed!
2023-12-16 10:48:07 INFO run_output: b'12.000000 2.000000\nTestcase 1 Failed!\n10.000000 10.000000\n480.000000 5.000000\nTestcase 3 Failed!\n1249975000.000000 1249975000.000000\n166150.000000 33230.000000\nTestcase 5 Failed!\nPassed 2/5 testcases!\nTest Failed!\n'


2023-12-16 10:48:07 INFO Index: 25...
2023-12-16 10:48:07 INFO Running task: 26_0...
2023-12-16 10:48:07 INFO Test Passed!


2023-12-16 10:48:07 INFO Index: 26...
2023-12-16 10:48:07 INFO Running task: 27_0...
2023-12-16 10:48:08 INFO Test Passed!


2023-12-16 10:48:08 INFO Index: 27...
2023-12-16 10:48:08 INFO Running task: 28_0...
2023-12-16 10:48:08 INFO Test Passed!


2023-12-16 10:48:08 INFO Index: 28...
2023-12-16 10:48:08 INFO Running task: 29_0...
2023-12-16 10:48:08 INFO Test Passed!


2023-12-16 10:48:08 INFO Index: 29...
2023-12-16 10:48:08 INFO Running task: 30_0...
2023-12-16 10:48:08 INFO Test Passed!


2023-12-16 10:48:08 INFO Index: 30...
2023-12-16 10:48:08 INFO Running task: 31_0...
2023-12-16 10:48:08 INFO Compilation Failed!
2023-12-16 10:48:08 INFO run_output: b'  File "/home/ubuntu/test_codellm/unit_tests/chatglm3/c++_python/test.py", line 71\n    zero_count = len_num - 1 if \\"0\\" in num else len_num\n                                 ^\nSyntaxError: unexpected character after line continuation character\n'


2023-12-16 10:48:08 INFO Index: 31...
2023-12-16 10:48:08 INFO Running task: 32_0...
2023-12-16 10:48:08 INFO Runtime Failed!
2023-12-16 10:48:08 INFO run_output: b"Exception: unsupported operand type(s) for +=: 'int' and 'str'\nException: unsupported operand type(s) for +=: 'int' and 'str'\nException: unsupported operand type(s) for +=: 'int' and 'str'\nException: unsupported operand type(s) for +=: 'int' and 'str'\nException: unsupported operand type(s) for +=: 'int' and 'str'\nPassed 0/5 testcases!\nTest Failed!\n"


2023-12-16 10:48:08 INFO Index: 32...
2023-12-16 10:48:08 INFO Running task: 33_0...
2023-12-16 10:48:09 INFO Test Passed!


2023-12-16 10:48:09 INFO Index: 33...
2023-12-16 10:48:09 INFO Running task: 34_0...
2023-12-16 10:48:09 INFO Test Passed!


2023-12-16 10:48:09 INFO Index: 34...
2023-12-16 10:48:09 INFO Running task: 35_0...
2023-12-16 10:48:09 INFO Test Passed!


2023-12-16 10:48:09 INFO Index: 35...
2023-12-16 10:48:09 INFO Running task: 36_0...
2023-12-16 10:48:09 INFO Test Passed!


2023-12-16 10:48:09 INFO Index: 36...
2023-12-16 10:48:09 INFO Running task: 37_0...
2023-12-16 10:48:09 INFO Test Passed!


2023-12-16 10:48:09 INFO Index: 37...
2023-12-16 10:48:09 INFO Running task: 38_0...
2023-12-16 10:48:09 INFO Test Passed!


2023-12-16 10:48:09 INFO Index: 38...
2023-12-16 10:48:09 INFO Running task: 39_0...
2023-12-16 10:48:09 INFO Test Passed!


2023-12-16 10:48:09 INFO Index: 39...
2023-12-16 10:48:09 INFO Running task: 40_0...
2023-12-16 10:48:09 INFO Test Passed!


2023-12-16 10:48:09 INFO Index: 40...
2023-12-16 10:48:09 INFO Running task: 41_0...
2023-12-16 10:48:09 INFO Test Passed!


2023-12-16 10:48:09 INFO Index: 41...
2023-12-16 10:48:09 INFO Running task: 42_0...
2023-12-16 10:48:10 INFO Runtime Failed!
2023-12-16 10:48:10 INFO run_output: b"Exception: name 'prev' is not defined\nException: name 'prev' is not defined\nException: name 'prev' is not defined\nException: name 'prev' is not defined\nException: name 'prev' is not defined\nPassed 0/5 testcases!\nTest Failed!\n"


2023-12-16 10:48:10 INFO Index: 42...
2023-12-16 10:48:10 INFO Running task: 43_0...
2023-12-16 10:48:10 INFO Test Passed!


2023-12-16 10:48:10 INFO Index: 43...
2023-12-16 10:48:10 INFO Running task: 44_0...
2023-12-16 10:48:10 INFO Test Passed!


2023-12-16 10:48:10 INFO Index: 44...
2023-12-16 10:48:10 INFO Running task: 45_0...
2023-12-16 10:48:10 INFO Test Passed!


2023-12-16 10:48:10 INFO Index: 45...
2023-12-16 10:48:10 INFO Running task: 46_0...
2023-12-16 10:48:10 INFO Test Passed!


2023-12-16 10:48:10 INFO Index: 46...
2023-12-16 10:48:10 INFO Running task: 47_0...
2023-12-16 10:48:10 INFO Test Passed!


2023-12-16 10:48:10 INFO Index: 47...
2023-12-16 10:48:10 INFO Running task: 48_0...
2023-12-16 10:48:11 INFO Test Passed!


2023-12-16 10:48:11 INFO Index: 48...
2023-12-16 10:48:11 INFO Running task: 49_0...
2023-12-16 10:48:11 INFO Test Passed!


2023-12-16 10:48:11 INFO Index: 49...
2023-12-16 10:48:11 INFO Running task: 50_0...
2023-12-16 10:48:11 INFO Compilation Failed!
2023-12-16 10:48:11 INFO run_output: b'  File "/home/ubuntu/test_codellm/unit_tests/chatglm3/c++_python/test.py", line 72\n    t := \\"\\"\n          ^\nSyntaxError: unexpected character after line continuation character\n'


2023-12-16 10:48:11 INFO Index: 50...
2023-12-16 10:48:11 INFO Running task: 51_0...
2023-12-16 10:48:11 INFO Test Passed!


2023-12-16 10:48:11 INFO Index: 51...
2023-12-16 10:48:11 INFO Running task: 52_0...
2023-12-16 10:48:11 INFO Test Passed!


2023-12-16 10:48:11 INFO Index: 52...
2023-12-16 10:48:11 INFO Running task: 53_0...
2023-12-16 10:48:11 INFO Test Passed!


2023-12-16 10:48:11 INFO Index: 53...
2023-12-16 10:48:11 INFO Running task: 54_0...
2023-12-16 10:48:11 INFO Test Passed!


2023-12-16 10:48:11 INFO Index: 54...
2023-12-16 10:48:11 INFO Running task: 55_0...
2023-12-16 10:48:13 INFO Test Passed!


2023-12-16 10:48:13 INFO Index: 55...
2023-12-16 10:48:13 INFO Running task: 56_0...
2023-12-16 10:48:18 INFO Test Passed!


2023-12-16 10:48:18 INFO Index: 56...
2023-12-16 10:48:18 INFO Running task: 57_0...
2023-12-16 10:48:18 INFO Test Passed!


2023-12-16 10:48:18 INFO Index: 57...
2023-12-16 10:48:18 INFO Running task: 58_0...
2023-12-16 10:48:18 INFO Test Passed!


2023-12-16 10:48:18 INFO Index: 58...
2023-12-16 10:48:18 INFO Running task: 59_0...
2023-12-16 10:48:23 INFO Test Passed!


2023-12-16 10:48:23 INFO Index: 59...
2023-12-16 10:48:23 INFO Running task: 60_0...
2023-12-16 10:48:23 INFO Test Failed!
2023-12-16 10:48:23 INFO run_output: b'[204.000000,205.000000,206.000000] [204.000000,205.000000,206.000000]\n[7.000000,3.000000,10.000000,8.000000,12.000000] [3.000000,7.000000,8.000000,10.000000,12.000000]\nTestcase 2 Failed!\n[3674.000000,3695.000000,3685.000000,3720.000000,3670.000000,3684.000000,3677.000000,3714.000000] [3670.000000,3674.000000,3677.000000,3684.000000,3685.000000,3695.000000,3714.000000,3720.000000]\nTestcase 3 Failed!\n[8587.000000,8609.000000] [8587.000000,8609.000000]\n[2451.000000,2483.000000,2447.000000,2438.000000,2488.000000,2437.000000,2510.000000,2466.000000,2473.000000,2457.000000,2515.000000,2444.000000,2442.000000,2458.000000,2489.000000,2517.000000,2518.000000,2479.000000] [2437.000000,2438.000000,2442.000000,2444.000000,2447.000000,2451.000000,2457.000000,2458.000000,2466.000000,2473.000000,2479.000000,2483.000000,2488.000000,2489.000000,2510.000000,2515.000000,2517.000000,2518.000000]\nTestcase 5 Failed!\nPassed 2/5 testcases!\nTest Failed!\n'


2023-12-16 10:48:23 INFO Index: 60...
2023-12-16 10:48:23 INFO Running task: 61_0...
2023-12-16 10:48:23 INFO Compilation Failed!
2023-12-16 10:48:23 INFO run_output: b'  File "/home/ubuntu/test_codellm/unit_tests/chatglm3/c++_python/test.py", line 79\n    possible_max = (sum_ - *b + m) % m\n                           ^\nSyntaxError: invalid syntax\n'


2023-12-16 10:48:23 INFO Index: 61...
2023-12-16 10:48:23 INFO Running task: 62_0...
2023-12-16 10:48:23 INFO Test Passed!


2023-12-16 10:48:23 INFO Index: 62...
2023-12-16 10:48:23 INFO Running task: 63_0...
2023-12-16 10:48:23 INFO Compilation Failed!
2023-12-16 10:48:23 INFO run_output: b'  File "/home/ubuntu/test_codellm/unit_tests/chatglm3/c++_python/test.py", line 77\n    return \\"IsFibo\\"\n            ^\nSyntaxError: unexpected character after line continuation character\n'


2023-12-16 10:48:23 INFO Index: 63...
2023-12-16 10:48:23 INFO Running task: 64_0...
2023-12-16 10:48:23 INFO Compilation Failed!
2023-12-16 10:48:23 INFO run_output: b'  File "/home/ubuntu/test_codellm/unit_tests/chatglm3/c++_python/test.py", line 91\n    print(f\\"[{l}, {r}]: {count}\\")\n            ^\nSyntaxError: unexpected character after line continuation character\n'


2023-12-16 10:48:23 INFO Index: 64...
2023-12-16 10:48:23 INFO Running task: 65_0...
2023-12-16 10:48:23 INFO Compilation Failed!
2023-12-16 10:48:23 INFO run_output: b'  File "/home/ubuntu/test_codellm/unit_tests/chatglm3/c++_python/test.py", line 72\n    if grid[k][j - 1] < grid[k][j] and grid[k][j] > grid[k][j + 1] and \\\\\n                                                                        ^\nSyntaxError: unexpected character after line continuation character\n'


2023-12-16 10:48:23 INFO Index: 65...
2023-12-16 10:48:23 INFO Running task: 66_0...
2023-12-16 10:48:23 INFO Test Passed!


2023-12-16 10:48:23 INFO Index: 66...
2023-12-16 10:48:23 INFO Running task: 67_0...
2023-12-16 10:48:24 INFO Test Passed!


2023-12-16 10:48:24 INFO Index: 67...
2023-12-16 10:48:24 INFO Running task: 68_0...
2023-12-16 10:48:24 INFO Compilation Failed!
2023-12-16 10:48:24 INFO run_output: b'  File "/home/ubuntu/test_codellm/unit_tests/chatglm3/c++_python/test.py", line 71\n    return \\"26.09.1918\\"\n            ^\nSyntaxError: unexpected character after line continuation character\n'


2023-12-16 10:48:24 INFO Index: 68...
2023-12-16 10:48:24 INFO Running task: 69_0...
2023-12-16 10:48:38 INFO Test Passed!


2023-12-16 10:48:38 INFO Index: 69...
2023-12-16 10:48:38 INFO Running task: 70_0...
2023-12-16 10:48:38 INFO Compilation Failed!
2023-12-16 10:48:38 INFO run_output: b'  File "/home/ubuntu/test_codellm/unit_tests/chatglm3/c++_python/test.py", line 70\n    hour = [\\"one\\", \\"two\\", \\"three\\", \\"four\\", \\"five\\", \\"six\\", \\"seven\\", \\"eight\\", \\"nine\\", \n             ^\nSyntaxError: unexpected character after line continuation character\n'


2023-12-16 10:48:38 INFO Index: 70...
2023-12-16 10:48:38 INFO Running task: 71_0...
2023-12-16 10:48:38 INFO Test Failed!
2023-12-16 10:48:38 INFO run_output: b'6.000000 2.000000\nTestcase 1 Failed!\n6.000000 2.000000\nTestcase 2 Failed!\n10.000000 4.000000\nTestcase 3 Failed!\n32912.000000 24008.000000\nTestcase 4 Failed!\n25786.000000 15986.000000\nTestcase 5 Failed!\nPassed 0/5 testcases!\nTest Failed!\n'


2023-12-16 10:48:38 INFO Index: 71...
2023-12-16 10:48:38 INFO Running task: 72_0...
2023-12-16 10:48:38 INFO Test Passed!


2023-12-16 10:48:38 INFO Index: 72...
2023-12-16 10:48:38 INFO Running task: 73_0...
2023-12-16 10:48:38 INFO Test Passed!


2023-12-16 10:48:38 INFO Index: 73...
2023-12-16 10:48:38 INFO Running task: 74_0...
2023-12-16 10:48:40 INFO Test Passed!


2023-12-16 10:48:40 INFO Index: 74...
2023-12-16 10:48:40 INFO Running task: 75_0...
2023-12-16 10:48:41 INFO Test Passed!


2023-12-16 10:48:41 INFO Index: 75...
2023-12-16 10:48:41 INFO Running task: 76_0...
2023-12-16 10:48:41 INFO Test Passed!


2023-12-16 10:48:41 INFO Index: 76...
2023-12-16 10:48:41 INFO Running task: 77_0...
2023-12-16 10:48:41 INFO Test Passed!


2023-12-16 10:48:41 INFO Index: 77...
2023-12-16 10:48:41 INFO Running task: 78_0...
2023-12-16 10:48:42 INFO Test Passed!


2023-12-16 10:48:42 INFO Index: 78...
2023-12-16 10:48:42 INFO Running task: 79_0...
2023-12-16 10:48:42 INFO Test Passed!


2023-12-16 10:48:42 INFO Index: 79...
2023-12-16 10:48:42 INFO Running task: 80_0...
2023-12-16 10:48:42 INFO Test Passed!


2023-12-16 10:48:42 INFO Index: 80...
2023-12-16 10:48:42 INFO Running task: 81_0...
2023-12-16 10:48:43 INFO Test Passed!


2023-12-16 10:48:43 INFO Index: 81...
2023-12-16 10:48:43 INFO Running task: 82_0...
2023-12-16 10:48:43 INFO Compilation Failed!
2023-12-16 10:48:43 INFO run_output: b'  File "/home/ubuntu/test_codellm/unit_tests/chatglm3/c++_python/test.py", line 74\n    s1[\\"1\\"] = 1\n        ^\nSyntaxError: unexpected character after line continuation character\n'


2023-12-16 10:48:43 INFO Index: 82...
2023-12-16 10:48:43 INFO Running task: 83_0...
2023-12-16 10:48:43 INFO Test Passed!


2023-12-16 10:48:43 INFO Index: 83...
2023-12-16 10:48:43 INFO Running task: 84_0...
2023-12-16 10:48:43 INFO Test Passed!


2023-12-16 10:48:43 INFO Index: 84...
2023-12-16 10:48:43 INFO Running task: 85_0...
2023-12-16 10:48:43 INFO Test Passed!


2023-12-16 10:48:43 INFO Index: 85...
2023-12-16 10:48:43 INFO Running task: 86_0...
2023-12-16 10:48:43 INFO Runtime Failed!
2023-12-16 10:48:43 INFO run_output: b'0.000000 5.000000\nTestcase 1 Failed!\n2.000000 2.000000\nException: list index out of range\nException: list index out of range\nException: list index out of range\nPassed 1/5 testcases!\nTest Failed!\n'


2023-12-16 10:48:43 INFO Index: 86...
2023-12-16 10:48:43 INFO Running task: 87_0...
2023-12-16 10:48:46 INFO Runtime Failed!
2023-12-16 10:48:46 INFO run_output: b"Exception: name 'ans' is not defined\nException: name 'ans' is not defined\nException: name 'ans' is not defined\nException: name 'ans' is not defined\nException: name 'ans' is not defined\nPassed 0/5 testcases!\nTest Failed!\n"


2023-12-16 10:48:46 INFO Index: 87...
2023-12-16 10:48:46 INFO Running task: 88_0...
2023-12-16 10:48:54 INFO Test Passed!


2023-12-16 10:48:54 INFO Index: 88...
2023-12-16 10:48:54 INFO Running task: 89_0...
2023-12-16 10:48:54 INFO Test Passed!


2023-12-16 10:48:54 INFO Index: 89...
2023-12-16 10:48:54 INFO Running task: 90_0...
2023-12-16 10:48:54 INFO Test Passed!


2023-12-16 10:48:54 INFO Index: 90...
2023-12-16 10:48:54 INFO Running task: 91_0...
2023-12-16 10:48:54 INFO Test Failed!
2023-12-16 10:48:54 INFO run_output: b'3.000000 3.000000\n2.000000 1.000000\nTestcase 2 Failed!\n2.000000 0.000000\nTestcase 3 Failed!\n5.000000 2.000000\nTestcase 4 Failed!\n2.000000 1.000000\nTestcase 5 Failed!\nPassed 1/5 testcases!\nTest Failed!\n'


2023-12-16 10:48:54 INFO Index: 91...
2023-12-16 10:48:54 INFO Running task: 92_0...
2023-12-16 10:48:55 INFO Test Passed!


2023-12-16 10:48:55 INFO Index: 92...
2023-12-16 10:48:55 INFO Running task: 93_0...
2023-12-16 10:48:55 INFO Test Passed!


2023-12-16 10:48:55 INFO Index: 93...
2023-12-16 10:48:55 INFO Running task: 94_0...
2023-12-16 10:48:55 INFO Test Passed!


2023-12-16 10:48:55 INFO Index: 94...
2023-12-16 10:48:55 INFO Running task: 95_0...
2023-12-16 10:48:55 INFO Test Passed!


2023-12-16 10:48:55 INFO Index: 95...
2023-12-16 10:48:55 INFO Running task: 96_0...
2023-12-16 10:48:55 INFO Test Passed!


2023-12-16 10:48:55 INFO Index: 96...
2023-12-16 10:48:55 INFO Running task: 97_0...
2023-12-16 10:48:55 INFO Test Passed!


2023-12-16 10:48:55 INFO Index: 97...
2023-12-16 10:48:55 INFO Running task: 98_0...
2023-12-16 10:48:55 INFO Test Passed!


2023-12-16 10:48:55 INFO Index: 98...
2023-12-16 10:48:55 INFO Running task: 99_0...
2023-12-16 10:48:55 INFO Test Passed!


2023-12-16 10:48:55 INFO Index: 99...
2023-12-16 10:48:55 INFO Running task: 100_0...
2023-12-16 10:48:56 INFO Test Passed!


2023-12-16 10:48:56 INFO [[1], [1], [1], [1], [1], [1], [1], [1], [2], [1], [1], [3], [1], [1], [1], [3], [1], [1], [1], [1], [1], [4], [4], [1], [4], [1], [1], [1], [1], [1], [2], [3], [1], [1], [1], [1], [1], [1], [1], [1], [1], [3], [1], [1], [1], [1], [1], [1], [1], [2], [1], [1], [1], [1], [1], [1], [1], [1], [1], [4], [2], [1], [2], [2], [2], [1], [1], [2], [1], [2], [4], [1], [1], [1], [1], [1], [1], [1], [1], [1], [1], [2], [1], [1], [1], [3], [3], [1], [1], [1], [4], [1], [1], [1], [1], [1], [1], [1], [1], [1]]
